src/openrct2/world/Sprite.h

```diff
@@ -193,7 +195,7 @@ enum
     SPRITE_FLAGS_PEEP_FLASHING = 1 << 9, // Deprecated: Use sprite_set_flashing/sprite_get_flashing instead.
 };
 
-enum
+enum LitterType : uint8_t
```

Comment:
Can you seize the opportunity and make this an `enum class`? If not, open an issue to track it and add to the corresponding project (on commit https://github.com/OpenRCT2/OpenRCT2/pull/13735/commits/617f6d5b1724341d7bf05a3059cc6fccb1db1230)

Reply:
I purposely left it for a follow on. Don't want to do everything in this pr